runbotics/runbotics-cli/src/utils/repository-path.ts

```diff
@@ -6,7 +6,7 @@ import { Emoji } from './emoji';
 const exec = promisify(nativeExec);
 
 const getRepositoryPath = async () => {
-    await exec('git config -l | grep remote.origin.url=https://github.com/runbotics/runbotics')
+    await exec('git config -l | grep -E "remote.origin.url=(https://github.com/runbotics/runbotics|git@github.com:runbotics/runbotics.git)"')
```

Comment:
@preludium e stands for **_enable_**.

Reply:
remove `.git` at the end of the line, its not necessary to clone a repo so there can be issues related to it